templates/netlify/README.md

```diff
@@ -30,14 +30,24 @@ netlify init
 
 ## Development
 
-The Netlify CLI starts your app in development mode, rebuilding assets on file changes.
+The Remix dev server starts your app in development mode, rebuilding assets on file changes.
```

Comment:
[sand] Minor addition to make the connection between `npm run dev` and `remix dev` clearer.
```suggestion
The Remix dev server starts your app in development mode, rebuilding assets on file changes. To start the Remix dev server:
```